pages/_app.js

```diff
@@ -57,7 +57,11 @@ class OpenCollectiveFrontendApp extends App {
         if (ctx.req) {
           scripts['google-maps'] = getGoogleMapsScriptUrl();
         } else {
-          await loadGoogleMaps();
+          try {
+            await loadGoogleMaps();
+          } catch (e) {
+            console.error(e);
+          }
```

Comment:
Sounds good, it's already better than what we have today.

Could be interesting to report the error to the user in the UI.

Reply:
Agree with the UI idea. I went for console instead of UI mostly for simplicity, but also it seems there is a precedent in the code for reporting to console when stripe is unavailable. Maybe a UI notice feature would incorporate maps, stripe, and others?

Also it might make sense to change the message if the script is blocked vs. otherwise unavailable, but I just wanted to fix the main problem.